package.json

```diff
@@ -13,8 +13,10 @@
     "@expo/react-native-action-sheet": "^4.0.1",
     "@expo/webpack-config": "^19.0.0",
     "@react-native-async-storage/async-storage": "1.18.2",
+    "@shopify/flash-list": "1.4.3",
```

Comment:
run `npm remove @shopify/flash-list` it is no longer needed in the project and has been causing errors on `main`